packages/admin-panel/src/index.js

```diff
@@ -6,15 +6,19 @@ import React, { lazy, Suspense } from 'react';
 import { BrowserRouter as Router, Route, Switch, Redirect } from 'react-router-dom';
 import { render as renderReactApp } from 'react-dom';
 import 'react-table/react-table.css';
+import { EnvBanner } from '@tupaia/ui-components';
 import AdminPanel from './App';
 import { AdminPanelProviders, VizBuilderProviders } from './utilities';
 import { Footer, Navbar } from './widgets';
 
 const VizBuilder = lazy(() => import('./VizBuilderApp'));
 
+const branch = process.env.REACT_APP_BRANCH;
```

Comment:
@reviewer Maybe the branch variable should just be moved into the `EnvBanner` component? What do you think?

Reply:
I agree, at least as a default (prop could still be exposed in case someone wanted to use a different variable, but not totally necessary)